### ISSUE TITLE
Outputting RBF objects to text file

### ISSUE THREAD

**Issue by oguevremont** [optimad/bitpit#334]
Hello, 

Thank you for developping such a useful library. I'm currently trying to use bitpit to represent a levelset field using RBF, and to output the information of an interpolated RBF to use with another application.

The current idea would be to make a PR adding export/import RBF as text files directly. In the same or a subsequent PR, I could also add some tests regarding the [Issue #180](https://github.com/optimad/bitpit/issues/180). Do you believe this is viable?

Thank you

**Comment by kgkara** [comment]
Hello @oguevremont. Thank you very much for your interest in using and programming in bitpit as well as for appreciating the work behind it. Your ideas seem very interesting. Since bitpit provides support for RBF, indeed you could use the existing classes  for your project, so that you can represent levelset fields with RBF. Some time will be needed before sharing with you some ideas/directions on this as well as the issue #180. So, soon enough there will be more news here. Thank you very much again.

**Comment by kgkara** [comment]
Hello @oguevremont. In case you prepared an initial implementation, we can start discussing the next steps and open a pull request.

**Comment by oguevremont** [comment]
Hello @kgkara, I have prepared an initial implementation for this, and it works pretty well. The library that we develop uses the RBF information to represent complex solids, and we solve the flow around these objects using FEM (https://github.com/lethe-cfd/lethe). It is still a work in progress.

On my bitpit fork, the script is located in /examples/levelset_prototype for now. I have also added a dependency to the Eigen library, to be able to use a sparse solver for the system of equations. I'll clean the code a bit and I would open a PR in the next few weeks.
